Makefile

```diff
@@ -49,21 +51,24 @@ clone-tuned:
 	  cd $(TUNED_DIR) && git checkout $(TUNED_COMMIT) && cd .. && \
 	  rm -rf $(TUNED_DIR)/.git)
 
-build: $(BINDATA) pkg/generated
+build: $(BINDATA) generate-deepcopy pkg/generated
 	$(GO_BUILD_RECIPE)
 	ln -sf $(PACKAGE_BIN) $(OUT_DIR)/openshift-tuned
 
 $(BINDATA): $(GOBINDATA_BIN) $(ASSETS)
 	$(GOBINDATA_BIN) -mode 420 -modtime 1 -pkg manifests -o $(BINDATA) assets/...
 	gofmt -s -w $(BINDATA)
 
-pkg/generated: $(API_TYPES)
+generate-deepcopy:
```

Comment:
This change basically makes the reason for using `Makefile` useless.  Please keep the dependencies.

Reply:
why is that ? this will re generated the zz deep copy files in case the api/types.go for the types we have are changed.   Is this handled somewhere else?
notice that `pkg/generated `was moved to line 71

Reply:
ok.  I now see I removed $(API_TYPES) by accident in the `pkg/generated:` section